qutip/models/tests/test_dicke.py

```diff
@@ -0,0 +1,309 @@
+"""
+Tests for Permutation Invariance methods
```

Comment:
QuTiP license stuff needs to be included at the top of the file. You can add yourselves to the list of authors for these files